src/postprocess/apirequests.py

```diff
@@ -103,7 +106,7 @@ def get_users_by_usernames(api, usernames):
         {
             "paging": False,
             "filter": "userCredentials.username:in:[%s]" % ",".join(usernames),
-            "fields": ":all,userCredentials[:all,userRoles[id,name]]",
+            "fields": "*,userCredentials[*,userRoles[id,name]]",
```

Comment:
Both `:all` and `*` are equivalent, right?

Reply:
i think so but not sure really, 
I always use *, i througth that  i have changed it to standarize, why use ":all" having "*" ? more short, without strange token ":".
But maybe in d2api and other apps you are using :all as standar..

Reply:
please @idelcano revert this change :)